cmd/ipfs-cluster-service/daemon.go

```diff
@@ -97,10 +107,11 @@ func createCluster(
 	ctx context.Context,
 	c *cli.Context,
 	cfgs *cfgs,
+	ident *identity.Identity,
 	raftStaging bool,
 ) (*ipfscluster.Cluster, error) {
 
-	host, pubsub, dht, err := ipfscluster.NewClusterHost(ctx, cfgs.clusterCfg)
+	host, pubsub, dht, err := ipfscluster.NewClusterHost(ctx, cfgs.clusterCfg, ident)
```

Comment:
Nitpick: could we pass identity before configs everywhere? Maybe it's just a personal taste, but it seems like identity is a more critical parameter than cfgs.